lib/rucio/core/rule.py

```diff
@@ -1424,14 +1425,13 @@ def reduce_rule(rule_id, copies, exclude_expression=None, session=None):
 
 
 @transactional_session
-def move_rule(rule_id, rse_expression, activity=None, source_replica_expression=None, session=None):
+def move_rule(rule_id: str, rse_expression: str, override: Dict[str, Any] = {}, session=None):
```

Comment:
Following today’s meeting and the mention of pitfalls when using mutable defaults, is this a concern?

Reply:
Could you elaborate on that? The `override` option is never changed...

Reply:
It’s not changed in this piece of code, but given the potential havoc it could cause in other parts of the Rucio source code (or, though highly unlikely, if this code is changed in the future) I think it’s a pattern that should be avoided completely. Pylint catches it as W0102.

(Context for anyone not present during the meeting: read [Python Mutable Defaults Are The Source of All Evil](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/))